004-governance.Rmd

```diff
@@ -0,0 +1,153 @@
+---
+output: github_document
+---
+
+# Tidyup 4: governance model
+
+**Champion**: Hadley Wickham  
+**Co-champion**: Tracy Teal  
+**Status**: Proposa
+l
+
+## Abstract
+
+Our open source packages currently don't have a well defined governance model.
+We do have a basic set of assumptions under which we operate, but we have not previously made them precise.
+This tidyup lays out a model that strikes a balance between the benevolent dictator and rough consensus models, and defines the relationships between four key roles: a large community of package **users,** a smaller pool of GitHub **contributors**, a team of **authors**, and a single **maintainer**.
+
+## Motivation
+
+We currently have one package (ggplot2) with a [formal governance model](https://github.com/tidyverse/ggplot2/blob/master/GOVERNANCE.md), and an informal, undocumented model that we've used for other packages (e.g. dtplyr, dbplyr).
+The goal of this tidyup is to come up with holistic, flexible model, that can become our default governance model going forward.
+This is part of a general movement to get better at defining and documenting our processes to so that there's a clear path from package user to package developer.
```

Comment:
```suggestion
This is part of a general movement to get better at defining and documenting our processes so that there's a clear path from package user to package developer.
```

Reply:
Thanks! (merged manually)